design/vela-core/componenttrait-composing-and-trait-CR-naming.md

```diff
@@ -0,0 +1,124 @@
+# ComponentTrait Composing and Trait CR Naming
+
+* Owner: Jianbo Sun (@wonderflow), Zhou Zheng Xi (@zzxwill)
+* Reviewers: KubeVela/Crossplane Maintainers
+* Status: Draft
+
+## Background
+
+With the evolution of TraitDefinition, definition name can be short and simple without group and kind information,
```

Comment:
```suggestion
Now definition name is no longer coupled with CRD name, it's align to capability name  in KubeVela,
```

Reply:
Professional! Changing `it's align to` to `it's aligned with` might be better.